packages/pwa-kit-dev/src/configs/webpack/plugins.js

```diff
@@ -7,25 +7,30 @@
 import webpack from 'webpack'
 import path, {resolve} from 'path'
 import glob from 'glob'
-import minimatch from 'minimatch'
 
 const projectDir = process.cwd()
+// eslint-disable-next-line @typescript-eslint/no-var-requires
 const pkg = require(resolve(projectDir, 'package.json'))
 
 const OVERRIDES_EXTENSIONS = '.+(js|jsx|ts|tsx|svg|jpg|jpeg)'
```

Comment:
Open question... Is there a way to make this plugin work without having to define a set list of file extensions that can be overridden? I'm thinking of a scenario where maybe we add a feature to the template what includes adding a code file or asset file with a new file type. The implementer of that that feature will have to know about extensibility and update this code as to ensure that the feature they added is extendable.

Reply:
The risk in moving away from an allow-list for file extensions is that customers are then opening to being capable of foot-gunning themselves by putting `routes.nope` as a file that will then successfully override `routes.jsx`